qiita_db/util.py

```diff
@@ -2311,3 +2317,476 @@ def send_email(to, subject, body):
         raise RuntimeError("Can't send email!")
     finally:
         smtp.close()
+
+
+def resource_allocation_plot(file, cname, sname, COL_NAME):
+    """Builds resource allocation plot for given filename and jobs
```

Comment:
Similar to \_df, why is COL_NAME in all caps?

Reply:
COL_NAME was defined as global constant before being added as an argument to functions. Changed to col_name